app/views/catalog/_index_list_default.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <% doc_presenter = index_presenter(document) %>
 
-<% if(doc_presenter.field_value('has_model_ssim') == 'Collection') %>
+<% if field_value_for(doc_presenter, 'has_model_ssim') == 'Collection' %>
```

Comment:
Should we add a conditional here? Is this method available in the production code?

Reply:
From the specs, I observed that this doesn't break anything in the current Blacklight but I can wrap it just in case